### ISSUE TITLE
Easy cleanup

### ISSUE THREAD

**Issue by michaelwisely** [brhoades/grader#1]
It'd be dandy if there was a way to wipe out all traces of assignment containers. That way, we have the option to start fresh whenever we have a new assignment to grade.

**Comment by brhoades** [comment]
I agree on this. I'll change the title of this to something more specific.

Let's add something a la:

    $ python grade.py remove hw5
    Are you sure you would like to remove all 27 containers for hw5 [y/n] (n):

Also:

     $ python grade.py remove hw5 bjrq48
     Removing container for bjrq48 hw5
